src/modules/nft/presentation/controllers/NFTController.stub.ts

```
@@ -1,37 +1,20 @@
 import { Request, Response } from "express";
+import { Request, Response } from 'express';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Request'. Consider to delete it or rename it.

'Request' is defined here:


(lint/suspicious/noRedeclare)